examples/jsm/loaders/NRRDLoader.js

```diff
@@ -419,9 +419,7 @@ class NRRDLoader extends Loader {
 		const spacingZ = new Vector3().fromArray( headerObject.vectors[ 2 ] ).length();
 		volume.spacing = [ spacingX, spacingY, spacingZ ];
 
-
 		// Create IJKtoRAS matrix
```

Comment:
Shouldn't this comment be moved too?

Reply:
I think no, because `transitionMatrix` is a part of building IJKtoRAS matrix.